modules/admin_manual/pages/configuration/server/email_configuration.adoc

```diff
@@ -78,8 +78,17 @@ options in one place, in your mail server configuration.
 === Setting Mail Server Parameters in config.php
 
 If you prefer, you may set your mail server parameters in
-`config/config.php`. The following examples are for SMTP, PHP, Sendmail,
-and Qmail.
+`config/config.php`. 
+
+=== Supported SMTP sending modes 
+
+* xref:configuration/server/email_configuration.html#smtp[SMTP]
+* xref:configuration/server/email_configuration.html#php-mail[PHP Mail]
+* xref:configuration/server/email_configuration.html#sendmail[Sendmail]
+* xref:configuration/server/email_configuration.html#qmail[Qmail]
+
+IMPORTANT: Compatibility of sending modes might depend on the installation environment. 
+In case of problems with sending mode it is recommended to try other mode configurations.
```

Comment:
```suggestion
IMPORTANT: Compatibility of _sending modes_ might depend on the installation environment. 
In case of problems with a _sending mode_, it is recommended to try other mode configurations.
```